src/client/containers/ConfigurationCreateContainer/hooks/useConfigurationCreate.ts

```diff
@@ -0,0 +1,99 @@
+import { useEffect } from "react";
+import { useMutation } from "react-query";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+import { requester } from "../../../lib";
+import { Resource, ResourceType, WebhookEventType } from "../../../constants";
+import { errorToText } from "./errorToText";
```

Comment:
`errorToText` modules (embedded below) need to be reworked:
 - Make sure that they contain only the relevant error messages to transform
 - Correct wording: For example, `Only project members can update integrations settings.` should be rephrased for `ConfigurationCreateContainer`, i.e., `Only project members can setup integrations.` etc.


https://github.com/zeplin/microsoft-teams-app/blob/2b57ed80561a048ab1bb9f2c32d77f586deb7a03/src/client/containers/ConfigurationCreateContainer/hooks/errorToText.ts#L12-L36

https://github.com/zeplin/microsoft-teams-app/blob/2b57ed80561a048ab1bb9f2c32d77f586deb7a03/src/client/containers/ConfigurationUpdateContainer/hooks/errorToText.ts#L12-L36

Reply:
👌